config/locales/notices/en.yml

```diff
@@ -16,25 +16,24 @@ en:
       resend_email_success: We have resent your password reset email.
     password_changed: You changed your password.
     resend_confirmation_email:
-      success: We have resent your confirmation email.
+      success: We resent your confirmation email.
```

Comment:
@esgoodman Know we don't intend to come across angrily to users --- LOL!--- but if we could change the verb/copy, that would be great!